pkg/manager-server/file_system_api.go

```diff
@@ -212,6 +212,15 @@ type FileSystemOemLvm struct {
 
 	// The lvcreate commandline, minus the "lvcreate" command.
 	LvCreate string `json:"LvCreate,omitempty"`
+
+	// The vgchange commandline, minus the "vgchange" command.
+	VgChange string `json:"VgChange,omitempty"`
+	
+	// The vgremove commandline, minus the "vgremove" command.
+	VgRemove string `json:"VgRemove,omitempty"`
```

Comment:
I finally found a vgremove manpage that shows `--nolocking` and `--lockopt`.  Are those what you wanted?

Most of the vgremove manpages I've found have only a few uninteresting options in them.

Reply:
Any TOSS instance should have the man pages you're looking for.

> I finally found a vgremove manpage that shows --nolocking and --lockopt. Are those what you wanted?

yes, precisely. It's the error snippet I provided in slack - reproduce here for others
```
command="vgremove --yes default-nate-2-0-xfs-0-0_vg" error="Error Running Command 'vgremove --yes default-nate-2-0-xfs-0-0_vg', StdErr:   Start a lock manager, lvmlockd did not find one running.\n  Global lock failed: check that global lockspace is started\n"
```